recipes/gawk/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "gawk" %}
+{% set version = "4.2.0" %}
+{% set sha256 = "c88046c6e8396ee548bcb941e16def809b7b55b60a1044b5dd254094f347c7d9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - win.gawkmisc.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - m2w64-toolchain  # [win]
+    - python  # [win]
```

Comment:
Of course, it would be better if I had awk around to rewrite my awk... or even sed...